plugin.py

```diff
@@ -4,10 +4,9 @@
 import sublime
 
 from LSP.plugin.core import logging
-from LSP.plugin.core.handlers import LanguageHandler
-from LSP.plugin.core.settings import read_client_config
+from LSP.plugin.core.sessions import AbstractPlugin
```

Comment:
You are supposed to import from the public export only. Anything else can result in breaking without notice.
```suggestion
from LSP.plugin import AbstractPlugin
```